frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tsx

```diff
@@ -141,6 +141,30 @@ const TAGGED_TEMPLATES = {
       </>
     );
   },
+  mdmEnrolled: (activity: IActivity) => {
+    return (
+      <>
+        An end user turned on MDM features for a host with serial number
+        <b>
+          {activity.details?.host_serial} (
+          {activity.details?.installed_from_dep ? "automatic" : "manual"})
```

Comment:
@roperzh installed_from_dep key means it was automatically installed correct?

Reply:
that's correct 👍

Reply:
@roperzh To test this, I should be able to removing my MDM, adding my laptop as a host, add MDM enrollment, and then see it in the activity feed, yes?

Reply:
You don't even have to add your laptop as a host using osquery, just enrolling into MDM (either manual or DEP) should generate the activity log.

I'd recommend using a VM, feel free to ping me if I can help!

Reply:
@RachelElysia I'd recommend adding tests for these new activity types in `ActivityItem.tests.tsx`. They are pretty quick to write and will allow us to test the text rendered based on the enrollment type.